doc/changes/changes_11.0.2.md

```diff
@@ -1,12 +1,15 @@
-# Virtual Schema Common JDBC 11.0.2, released 2023-??-??
+# Virtual Schema Common JDBC 11.0.2, released 2023-09-15
 
-Code name:
+Code name: fix in jdbc connection management.
```

Comment:
```suggestion
Code name: Fix in JDBC connection management
```
This will become the title of the GitHub release, e.g. https://github.com/exasol/virtual-schema-common-jdbc/releases/tag/11.0.1